.changeset/metal-hornets-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend-module-pinniped-provider': patch
+---
+
+Introduced PinnipedStrategyCache to act as a metadata cache for the PinnipedAuthenticator.
```

Comment:
```suggestion
Introduced metadata cache for the `pinniped` provider.
```

Just removing some internal names from the changeset, exact implementation isn't too important 😁